homes/models.py

```diff
@@ -0,0 +1,154 @@
+from django.db    import models
+
+class HomeType(models.Model):
+    name = models.CharField(max_length=32)
+
+    class Meta:
+        db_table = "home_types"
+
+class BuildingType(models.Model):
+    name = models.CharField(max_length=32)
+
+    class Meta:
+        db_table = "building_types"
+
+class Region(models.Model):
+    name                  = models.CharField(max_length=16)
+    latitude              = models.FloatField()
+    longitude             = models.FloatField()
```

Comment:
넵 ! 
:+1:

Reply:
아하... 블로그 글을 보니 
>Float is generally an approximation, see here for some simple examples. You could get very nice results modifying your model to something like DecimalField(max_digits=9, decimal_places=6), since decimals are very important in coordinates but using more than 6 is basically meaningless.
[스택오버플로우 링크](https://stackoverflow.com/questions/30706799/which-model-field-to-use-in-django-to-store-longitude-and-latitude-values)
float보다는 decimal 이 더 편하다고해서 decimal로 수정하겠습니다.

Reply:
확인했습니다!